libraries/core_libs/consensus/include/final_chain/state_api_data.hpp

```diff
@@ -14,13 +14,11 @@ namespace taraxa::state_api {
 
 struct TaraxaEVMError : std::runtime_error {
   std::string const type;
-  TaraxaEVMError(std::string type, std::string msg);
-  ~TaraxaEVMError() throw();
```

Comment:
are you sure about this? Did you try to cause this exception?

Reply:
this throw() is not actual exception throwing, it tells the compiler that the destructor will not throw... Automatically generated one does the same thing